### ISSUE TITLE
Cannot read property 'clean' of null

### ISSUE THREAD

**Issue by sylido** [Meteor-Community-Packages/meteor-autoform#1257]
In the function `validateFormDoc` the line that calls the`clean` function is acting weird.
`ss.clean(docForValidation, {` For some reason `ss` seems to be `null`.

This happens only on one form which is of type "method", there is a collection and schema that are defined for it. Any ideas appreciated.

**Comment by lukiies** [comment]
yes.
i have the same issue. if You define parameter collection and schema together but schema isn't attached to this collection - if think there is a bug. Sample situation is when you want to edit users data without redefining default users and login module schema.

**Comment by serut** [comment]
Same issue for me, I don't know the cause but it blocks the auto refresh.
I rebooted and things get back to work.

**Comment by Rathius** [comment]
this is still an issue but the error I received is "ss not defined"

**Comment by harveysanders** [comment]
For reference:
https://github.com/Meteor-Community-Packages/meteor-autoform/issues/1248
TLDR:
As @lukiies stated, if you use `type=method` _and_ `collection=SomeCollection`, you need to have a schema attached to the collection, `SomeCollection`. If you don't have a schema attached to the schema, remove the `collection` option from the form.
